### ISSUE TITLE
Event input

### ISSUE THREAD

**Issue by ramya6016** [Deimos-Applications/dialog_flowtter#17]
When i try to raise event input from dialog_flowtter, it shows field input not set. Please help me.

```
DetectIntentResponse response = await dialogFlowtter.detectIntent(
      queryInput: QueryInput(
        eventInput: EventInput.fromJson(event),
      ),
      audioConfig: OutputAudioConfig(),
    );
```

```
E/flutter (17956): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: Exception: INVALID_ARGUMENT: Field input not set., (400)
E/flutter (17956): #0      DialogFlowtter.detectIntent
package:dialog_flowtter/src/dialog.dart:209
E/flutter (17956): <asynchronous suspension>
E/flutter (17956): #1      _HomeScreenState.setDialog
package:email_password_login/screens/home_screen.dart:282
E/flutter (17956): <asynchronous suspension>
```

**Comment by lawcha** [comment]
Hello,
I made a PR [here](https://github.com/Deimos-Applications/dialog_flowtter/pull/18) fixing this issue.

The problem comes from the "eventInput" parameter which is not named correctly (see https://cloud.google.com/dialogflow/es/docs/reference/rest/v2beta1/QueryInput).

**Comment by ElZombieIsra** [comment]
Can you guys confirm this issue was solved with #18 using version `0.3.2`?

**Comment by lawcha** [comment]
When I cloned the repo yesterday, the last tag was the version 0.3.1.
